snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -64,7 +86,25 @@ def destroy(self) -> None:
 
         if not self._instance_info.is_stopped():
             self._multipass_cmd.stop(instance_name=self.instance_name)
-        self._multipass_cmd.delete(instance_name=self.instance_name)
+        if self._is_ephemeral:
+            self.clean_project()
+
+    def mount_project(self) -> None:
+        # Resolve the home directory
+        home_dir = (
+            self._multipass_cmd.execute(
+                command=["pwd"], hide_output=True, instance_name=self.instance_name
```

Comment:
This seems fragile. Would something like `command=["printenv", "HOME"]` be better?